modules/apps/transfer/ibc_module_test.go

```diff
@@ -103,7 +110,11 @@ func (suite *TransferTestSuite) TestOnChanOpenInit() {
 
 			if tc.expPass {
 				suite.Require().NoError(err)
-				suite.Require().Equal(types.Version, version)
+				if v1 {
+					suite.Require().Equal(types.Version1, version)
+				} else {
+					suite.Require().Equal(types.Version, version)
+				}
```

Comment:
I wonder if we should bother adding the `v1` bool to the test func and instead just do `suite.Require().Contains(types.SupportVersions, version)` 

might be cleaner.

`ibctesting.NewTransferPath()` uses `transfertypes.Version` so will now default to ics20-2, right? 

Maybe we should malleate cases like `"success: ics20-1 legacy"` for each test func where we set it back to ics20-1

Reply:
can also use `expVersion`